packages/headless/package.json

```diff
@@ -79,5 +79,8 @@
     "rollup-plugin-terser": "7.0.2",
     "ts-jest": "26.5.6",
     "ts-node": "10.2.1"
+  },
+  "engines": {
+    "node": ">=10"
```

Comment:
I needed to set a minimum of node 10 after adjusting to `UrlSearchParams` 🤞
Before this change, the project supported a minimum of node 8.